shaka/src/mapping/v8/js_wrappers.cc

```diff
@@ -238,18 +238,19 @@ bool IsDerivedFrom(BackingObject* ptr, const std::string& name) {
 
 bool RunScript(const std::string& path) {
   util::FileSystem fs;
-  std::vector<uint8_t> source;
-  CHECK(fs.ReadFile(path, &source));
+  std::string source;
+  CHECK(fs.ReadFile(path, source));
   v8::Local<v8::String> code =
       v8::String::NewFromUtf8(GetIsolate(),
-                              reinterpret_cast<const char*>(source.data()),
-                              v8::NewStringType::kNormal, source.size())
+                              source.c_str(),
+                              v8::NewStringType::kNormal
+                              source.length())
           .ToLocalChecked();
```

Comment:
Not sure if any additional work is required here to ensure that unicode characters are properly decoded.